fastchat/serve/cli.py

```diff
@@ -10,9 +10,11 @@
 
 from fastchat.conversation import conv_templates, SeparatorStyle
 
+from fastchat.serve.shared_model_utils import tokensByDevice
+
 
 @torch.inference_mode()
-def generate_stream(tokenizer, model, params, device,
+def generate_stream(tokenizer, model, params, device, debug,
```

Comment:
regarding PR #195 
Couldn't some of the issues you mentioned like the 'loading shards' be attributed to this debug addition?

Reply:
nope